<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllergyDialog</class>
 <widget class="QDialog" name="AllergyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AllergyDialog</string>
  </property>
  <widget class="QGroupBox" name="typeGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>208</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Тип</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="allergyRadio">
      <property name="text">
       <string>Алергия</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="intoleranceRadio">
      <property name="text">
       <string>Непоносимост</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="categoryCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Към храна</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Към медикамент</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Към околна среда</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Към биологичен материал</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="criticalityGroup">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>133</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Критичност</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="lowRiskRadio">
      <property name="text">
       <string>Нисък риск</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="highRiskRadio">
      <property name="text">
       <string>Висок риск</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="undifinedRiskRadio">
      <property name="text">
       <string>Неопределен</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="clinicalCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Активен</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Активен (повторно проявяване)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Активен (в рецидив)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Неактивен</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Неактивен (в ремисия)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Неактивен (решен)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Клиничен статус:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Верификационен статус:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="verificationCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Непотвърден</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Непотвърден - условно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Непотвърден - противоречиво</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Потвърден</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Опроверган</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Погрешно въведен</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание:</string>
   </property>
  </widget>
  <widget class="CustomDateEdit" name="dateEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="displayFormat">
    <string notr="true">d.M.yyyy 'г.'</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="dateCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата на последно възникване:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomDateEdit</class>
   <extends>QDateEdit</extends>
   <header>View/uiComponents/customdateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
